frb_codegen/src/binary/commands.rs

```diff
@@ -172,6 +172,10 @@ pub(crate) struct GenerateCommandArgsPrimary {
     /// Dump all internal data. Same as `--dump` with all possible choices chosen.
     #[arg(long)]
     pub dump_all: bool,
+
+    /// List of cargo feature flags to enable when generating
+    #[arg(long)]
+    pub features: Option<Vec<String>>,
```

Comment:
nit: Shall we rename it to something like `rust_features`? For example, suppose one day in the future Dart supports features as well, then `features` may be confusing while `rust_features` is more understandable.